package.json

```diff
@@ -47,6 +47,7 @@
     "eslint-plugin-jsx-a11y": "6.1.2",
     "eslint-plugin-react": "7.11.1",
     "eslint-plugin-wordpress": "git://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress.git#1774343f6226052a46b081e01db3fca8793cc9f1",
+    "har-validator": "^5.1.3",
```

Comment:
This fixes an issue downloading `har-validator@5.1.2`, because that package at that version doesn't exist anymore 🤷🏼‍♀️

Reply:
npm shenanigans!